policyengine_us/parameters/gov/states/oh/tax/income/credits/lump_sum_distribution/age_threshold.yaml

```diff
@@ -0,0 +1,10 @@
+description: Ohio limits the lump sum distribution credit to filers of this age or older.
+values:
+  2021-01-01: 65
+metadata:
+  unit: year
+  period: year
+  label: Ohio Lump Sum Distribution Credit age threshold
+  reference:
+    - title: Ohio 2021 Instructions for Filing Original and Amended Individual Income Tax (IT 1040) and  School District Income Tax (SD 100)
```

Comment:
add the law as a reference here: 
https://law.justia.com/codes/ohio/2022/title-57/chapter-5747/section-5747-055/